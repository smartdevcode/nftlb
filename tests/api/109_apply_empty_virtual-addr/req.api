VERB="POST"
URI="farms"
DESC="Send empty farms... "
FILE="data.json"
