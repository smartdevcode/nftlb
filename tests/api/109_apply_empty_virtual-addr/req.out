{"response": "error parsing buffer. Key 'virtual-addr' cannot be empty"}